enhancements/freakfortress/firefixfreakfortress.sp

```diff
@@ -0,0 +1,52 @@
+#include <sourcemod>

```

Comment:
This file name `firefixfreakfortress.sp` should match the compiled plugin which is `firefix_ff_2` on the server. I suggest `firefix_ff2` to be consistent with how other FF2 plugins are named.